### PR TITLE
add metrics gapped by 5s 

### PR DESCRIPTION
**Pull request by knightXun**
I add some metrics gapped by 5s, So prometheus can collect metrics in every 5s.

### CODE REVIEW

File: src/common/stats/StatsManager.cpp [hunk 1]
```diff
@@ -86,7 +87,7 @@ int32_t StatsManager::registerHisto(folly::StringPiece counterName,
                 bucketSize,
                 min,
                 max,
-                StatsType(60, {seconds(60), seconds(600), seconds(3600)}))));
```
Comment (laura-ding) [inline src/common/stats/StatsManager.cpp]: Why change `numBuckets` to `720`, `numBuckets` is each level

Comment (knightXun) [reply src/common/stats/StatsManager.cpp]: i thought `numBuckets` means time gap, `60` means statsmanager scrape metrics every minute. If we want to scape metrics every 5s, It needs 60 * 12 buckets.

**Comment by knightXun** [comment]
ping @laura-ding

**Comment by laura-ding** [comment]
jenkins go

**Comment by knightXun** [comment]
jenkins go

**Comment by knightXun** [comment]
ping @dutor

**Comment by Shylock-Hg** [comment]
The duration record the metrics don't matter to prometheus scrape cycle.The folly statistics already handle time, so the metrics expose to pometheus or other metrics collector don't need to concern `time`.

**Comment by knightXun** [comment]
@Shylock-Hg folly store last minute metrics. For example, Promethues will get some result in 1s and 59s. The time gap is too large. It is better shrink one minute to 5 seconds. Prometheus can scrape metrics every 5 seconds.

**Pull request merged**